js/wysiwyg/crm.ckeditor.js

```diff
@@ -68,6 +68,11 @@
 
       $(item).addClass('crm-wysiwyg-enabled');
 
+      var isFullPage = false;
+      if ($(item).hasClass('crm-wysiwyg-fullpage')) {
+        isFullPage = true;
+      }
```

Comment:
These 4 lines could be condensed to `var isFullPage = $(item).hasClass('crm-wysiwyg-fullpage');`